PROV/prov-cco-directmappings.ttl

```diff
@@ -51,41 +52,41 @@ prov:wasAssociatedWith    rdfs:subPropertyOf cco:has_agent ;
 prov:wasInvalidatedBy     rdfs:subPropertyOf cco:is_affected_by ;
                           rdfs:comment "An Entity that was invalidated by an Activity is also affected by that Activity."@en .
 prov:wasGeneratedBy       rdfs:subPropertyOf cco:is_output_of ;
-                          rdfs:comment "An Entity that was generated by an Activity is output of that Activity"@en .
+                          rdfs:comment "An Entity that was generated by an Activity is output of that Activity."@en .
 prov:wasInformedBy        rdfs:subPropertyOf cco:caused_by ;
-                          rdfs:comment "If the Activity A1 was informed by the Activity A2, then A1 is caused by A2"@en .
```

Comment:
But in your supposed counterexample, "I" is not an activity. So maybe today's weather (as a process?) does not cause me, but causes my knowledge of the weather (as a process)?

Reply:
Yeah, this one isn't very good, but I couldn't find any other close matches for relations between processes or occurrents. I'm fine with leaving this unmapped because there are others we definitely have no mappings for.